source/ns_cmdline.c

```diff
@@ -1172,13 +1174,15 @@ void cmd_char_input(int16_t u_data)
         cmd_escape_read(u_data);
         return;
     }
+    if (cmd.prev_ch == '\r' && u_data == '\n') {
```

Comment:
if we move this block before passthrough  check and change state that it can be true only once - doesn't it then cover all scenarios we want ?

Reply:
yeah - I think its enough if we add as an known issue that passthrough doesn’t get following `\n` if `\r` alone is used for line feeds.

Reply:
I will change implementation a bit and more tests for passthrough which covers these scenarios we have mentioned here..